features/tailormade_auth/src/main/java/com/future/tailormade_auth/core/model/request/SignInRequest.kt

```diff
@@ -0,0 +1,8 @@
+package com.future.tailormade_auth.core.model.request
+
+data class SignInRequest(
+
+    val username: String = "",
```

Comment:
for mandatory field, please remove the default value
it will useful in code thee feature and unit testing